core/src/main/java/org/apache/iceberg/TableMetadataParser.java

```diff
@@ -498,6 +498,7 @@ static TableMetadata fromJson(FileIO io, String metadataLocation, JsonNode node)
         entries.build(),
         metadataEntries.build(),
         refs,
+        ImmutableList.of(), /* TODO: support statistics */
```

Comment:
I think this should be done in a PR with the `TableMetadata` updates.

Reply:
moved to https://github.com/apache/iceberg/pull/5450